tests/crop_test.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+
+# Test setting the "preserve_ar" stream configuration flag
+
+import cv2
+
+from picamera2 import Picamera2
+
+picam2 = Picamera2()
+
+for m, l in [(False, False), (False, True), (True, False), (True, True)]:
+    cfg = picam2.create_video_configuration(main={"size": (1920, 1080), "format": 'XRGB8888', "preserve_ar": m},
+                                            lores={"size": (640, 640), "format": 'XRGB8888', "preserve_ar": l},
```

Comment:
Just wondering what this does on a Pi 4 where you can't have RGB on the lores?

Reply:
Good question - I assumed the validate() would change this to YUV and everything would be good.  But I've not explicitly tried it...

Actually, there's no reason lores needs to be RGB, perhaps I should just change it to YUV.